homeassistant/components/suez_water/quality_scale.yaml

```
@@ -0,0 +1,88 @@
+rules:
+  # Bronze
+  config-flow: todo
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
```

Why is `config-flow` `todo` and is this one `done`?

---

Because I forgot to mark it after @allenporter indicate that data description are missing in config flow.

```suggestion
  config-flow-test-coverage: todo
```